config.json

```
@@ -141,7 +141,24 @@
                 "MAX_LIMIT": 1,
                 "WINDOW_SIZE": 1440
             }
-        }
+        },
+        {
+            "ID": "Beam",
+            "NAME": "Beam Testnet",
+            "TOKEN": "MC",
+            "RPC": "https://subnets.avax.network/beam/testnet/rpc",
+            "CHAINID": 13337,
+            "EXPLORER": "https://subnets-test.avax.network/beam",
+            "IMAGE": "https://glacier-api.avax.network/proxy/chain-assets/main/chains/13337/chain-logo.png",
+            "MAX_PRIORITY_FEE": "3000000000",
+            "MAX_FEE": "100000000000",
+            "DRIP_AMOUNT": 2,
+            "DECIMALS": 18,
+            "RATELIMIT": {
+                "MAX_LIMIT": 1,
+                "WINDOW_SIZE": 1440
+            }
+        },
```

Can you remove the extra comma here?
```suggestion
        }
```

---

oops sorry, done now